sim/paladin/retribution/retribution.go

```diff
@@ -178,6 +147,135 @@ func (ret *RetributionPaladin) openingRotation(sim *core.Simulation) {
 	}
 }
 
-func (ret *RetributionPaladin) testingMechanics(sim *core.Simulation) {
+func (ret *RetributionPaladin) ActRotation(sim *core.Simulation) {
+	// Setup
+	target := sim.GetPrimaryTarget()
+
+	gcdCD := ret.GetRemainingCD(core.GCDCooldownID, sim.CurrentTime)
+	crusaderStrikeCD := ret.GetRemainingCD(paladin.CrusaderStrikeCD, sim.CurrentTime)
+	nextCrusaderStrikeCD := ret.CDReadyAt(paladin.CrusaderStrikeCD)
+	judgementCD := ret.GetRemainingCD(paladin.JudgementCD, sim.CurrentTime)
+
+	sobActive := ret.HasActiveAura(ret.SealOfBloodAura.Label)
```

Comment:
You can just do ```sobActive := ret.SealOfBloodAura.IsActive()```. Cleaner and faster!

Same for the line below.

Reply:
Used your recommended method of checking for the aura